modules/deepvariant/main.nf

```diff
@@ -0,0 +1,58 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+// TODO nf-core: A module file SHOULD only define input and output files as command-line parameters.
+//               All other parameters MUST be provided as a string i.e. "options.args"
+//               where "params.options" is a Groovy Map that MUST be provided via the addParams section of the including workflow.
+//               Any parameters that need to be evaluated in the context of a particular sample
+//               e.g. single-end/paired-end data MUST also be defined and evaluated appropriately.
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DEEPVARIANT {
+    tag "$meta.id"
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::deepvariant=1.1.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/deepvariant:1.1.0--py36hf3e76ba_2"
+    } else {
+        // TODO update the bioconda container to work with run_deepvariant.sh script
+        // container "quay.io/biocontainers/deepvariant:1.1.0--py36hf3e76ba_2"
+        container "google/deepvariant:1.1.0"
```

Comment:
I'm not familiar with this but is there a reason why you commented out the container hosted by quay.io? If you use the statement `google/deepvariant:1.1.0` should you include the host? `docker.io/google/deepvariant:1.1.0`

Reply:
The reason is that the quay version doesn't contain the new `/opt/deepvariant/bin/run_deepvariant` command which combines all other scripts.

Therefore, I was planning to get started with the official Google container and then transition to the `quay` one - what do you suggest?

Reply:
Update: 

Internally, the exact commands used via the `run_depvariant` wrapper seem to be the following (for the command mentioned [here](https://github.com/google/deepvariant/blob/r1.2/docs/deepvariant-quick-start.md#run-deepvariant-with-one-command))

```
time seq 0 1 | parallel -q --halt 2 --line-buffer /opt/deepvariant/bin/make_examples --mode calling --ref "/input/ucsc.hg19.chr20.unittest.fasta" --reads "/input/NA12878_S1.chr20.10_10p1mb.bam" --examples "/output/intermediate_results_dir/make_examples.tfrecord@2.gz" --gvcf "/output/intermediate_results_dir/gvcf.tfrecord@2.gz" --regions "chr20:10,000,000-10,010,000" --task {}

time /opt/deepvariant/bin/call_variants --outfile "/output/intermediate_results_dir/call_variants_output.tfrecord.gz" --examples "/output/intermediate_results_dir/make_examples.tfrecord@2.gz" --checkpoint "/opt/models/wgs/model.ckpt" --openvino_model_dir "/output/intermediate_results_dir"

time /opt/deepvariant/bin/postprocess_variants --ref "/input/ucsc.hg19.chr20.unittest.fasta" --infile "/output/intermediate_results_dir/call_variants_output.tfrecord.gz" --outfile "/output/output.vcf.gz" --nonvariant_site_tfrecord_path "/output/intermediate_results_dir/gvcf.tfrecord@2.gz" --gvcf_outfile "/output/output.g.vcf.gz"

```